src/parsing.jl

```diff
@@ -235,18 +235,46 @@ function Base.parse(::Type{ZonedDateTime}, pqv::PQValue{PQ_SYSTEM_TYPES[:timesta
     end
 
     for fmt in TIMESTAMPTZ_FORMATS[1:end-1]
-        try
-            return parse(ZonedDateTime, str, fmt)
-        catch
-            continue
-        end
+        parsed = tryparse(ZonedDateTime, str, fmt)
+        parsed !== nothing && return parsed
     end
     # Cut off digits after the third after the decimal point,
     # since DateTime in Julia currently handles only milliseconds, see Issue #33
```

Comment:
This issue was close 3 years ago?
https://github.com/JuliaTime/TimeZones.jl/issues/33

Reply:
https://github.com/invenia/LibPQ.jl/pull/119#discussion_r312218791